Libplanet/Blockchain/BlockChain.cs

```diff
@@ -604,8 +604,7 @@ bool renderActions
                 HashDigest<SHA256>? tip = Store.IndexBlockHash(Id, -1);
 
                 var nonceDeltas = new Dictionary<Address, long>();
-                var orderedTransactions = block.Transactions.OrderBy(tx => tx.Nonce);
-                foreach (Transaction<T> tx1 in orderedTransactions)
```

Comment:
It would be better it there's a comment that mentions `blcok.Transactions` are already sorted in the tx nonce order.

Reply:
I rebased and added comment